uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,64 @@
+var Promise = require('es6-promise').Promise;
```

Comment:
Everything related to leave and absence is in the same folder. Modal should not be an exception. I mean when I see a folder specifically for leaveandabsence I would expect all the components related to that inside.